frontend/src/components/Semester/SemesterForm/SemesterForm.js

```diff
@@ -141,7 +140,7 @@ const SemesterForm = (props) => {
                 onClose={closeDialogForGroup}
             />
             <form onSubmit={handleSubmit}>
-                <div className="semester-checkbox">
+                <div className="semester-checkboxes">
                     <div>
```

Comment:
do you need this empty div ?

Reply:
yes, I need it for styling fields in one row in centre